### ISSUE TITLE
Tengai Makyou Zero not working anymore

### ISSUE THREAD

**Issue by rayabusa** [dborth/snes9xgx#721]
Tengai Makyou Zero translated rom is not working like was before anymore, black screen after the initial tests

**Comment by askotx** [comment]
What did you do? Give more info, we don't know what you've done. If it was working, then you have changed some settings.

**Comment by rayabusa** [comment]
im using the 4.3.7 Commit.c9d0680, version, the version before it was working fine, now its not.

Didn't made any changes on settings, only updated the emulator.

**Comment by Nashismo** [comment]
Unless you compiled it yourself, the game was NOT working on official builds of this emu. I actually did it, and got "Tengai Makyou Zero translated" working, BUT, the sound issues got even WORSE than when using the 4.3.7 official version (which also has problems with sound).

Also I saw your issue that you opened below this one, if you where having problems with sound, this is why, the latest non public (compile yourself) builds are working terrible in terms of sound.
